src/views/demo-page/demo-page.js

```diff
@@ -75,70 +30,80 @@ customElements.define('demo-page', class extends AIHTMLElement {
           document.querySelector('ai-app-bar').shadowRoot.querySelector('slot[name=dropdowns]').assignedNodes().forEach(slot => slot.theme = slot.hasAttribute('theme') ? false : true);
         "
       >Toggle theme</button>
-
-      <!-- Table component -->
-      <div class="table-wrapper">
-        <table class="ai-table sticky">
-          <thead>
-            <tr>
-              ${headers.map(column => getCellMarkup('col', true, column))}
-            </tr>
-          </thead>
-          <tbody>
-            ${rows.map(row => html`
-              <tr>
-                ${row.map((column, columnIndex) => getCellMarkup('row', columnIndex === 0, column))}
-              </tr>
-            `)}
-          </tbody>
-        </table>
-      </div>
-
-      <!-- Diff component -->
-      <div class="sample diff">
-        <h3>ai-diff</h3>
-        <ai-diff text="Deprecated" current="true"></ai-diff><br>
-        <ai-diff text="Stability" old="Experimental" current="Stable"></ai-diff><br>
+    ${until(/* eslint-disable indent */
+        getChangesModel().then(model => model.getLayout().then((layout) => {
+            const data = layout.qHyperCube.qPivotDataPages[0]
+              ? layout.qHyperCube.qPivotDataPages[0] : null;
+            if (data) {
+              const tblHeaders = new Set(data.qTop.map(el => (el.qText || '')));
+              const rowHeaders = new Set(data.qLeft.map(el => (el.qText || '')));
+              return html`
+              <div class="table-wrapper">
+                <table class="ai-table sticky">
+                  <thead>
+                    <tr>
+                    <th></th>
+                    ${repeat(tblHeaders, null, header => html`<th>${header}</th>`)}
```

Comment:
we don't need repeat from lit-html, use regular array.map
```suggestion 
                    ${tblHeaders.map(header => html`<th>${header}</th>`)}
```

Reply:
unnecessary change, the demo page will go away anyway